tests/checker/__init__.py

```diff
@@ -210,16 +227,20 @@ def get_resultlines(self, filename):
         ignoring empty lines and lines starting with a hash sign (#).
         """
         resultfile = get_file("%s.result" % filename)
-        d = {'curdir': get_file_url(os.getcwd()),
-             'datadir': get_file_url(get_file()),
-            }
+        d = {
+            "curdir": get_file_url(os.getcwd()),
+            "datadir": get_file_url(get_file()),
+        }
         # the webserver uses the first free port number
-        if hasattr(self, 'port'):
-            d['port'] = self.port
+        if hasattr(self, "port"):
+            d["port"] = self.port
         # all result files are encoded in utf-8
         with codecs.open(resultfile, "r", "utf-8") as f:
```

Comment:
Noticed in passing, not part of this PR, but: with the Python 2 drop we can replace `codecs.open(..., "UTF-8")` with just `open(..., encoding="UTF-8")`.